tools/ansible/playbooks/bootstrap.yml

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2018 by Delphix. All rights reserved.
+#
+---
+- hosts: all
```

Comment:
can we add the `become: true` line here? rather than having it in the `apt` invocation? this way, if/when we add more tasks, we don't have to add `become: true` to all of them.